.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015"]
+}
```

Comment:
I'm ok with this, but I think that we already provide the babel config through the package.json file no ? Why would we need this ? (asking just in case we can avoid duplication)